web/src/components/questions/GenericQuestion.tsx

```diff
@@ -47,7 +47,9 @@ export default function GenericQuestion({
 
   return (
     <Popup isOpen aria-label={_("Question")}>
```

Comment:
As said, I expected a title={something}  here. 

```suggestion
    <Popup isOpen title={_("Package installation failed")}>
```

I know this is the GenericQuestion component, but if I'm not wrong we could create an specific component for "RetryPackageInstallationQuestion" question and the backend give a type for it. Do not remember the details form the top of my head.

Reply:
Yes, but I'd like to avoid a specific PackageQuestion. I think having a title make sense in many cases (all?) so I'd like to support optional title in the generic implementation.

Because extending the Questions API is not trivial (do not forget that questions can be handled using command line interface) I'll create a separate card for that.

Reply:
But the problem is that title is not exposed, but included in the text. It's not optimal IMHO and lead us to start adding workarounds not truly needed. 

Now we do not have actually a title, but just a short sentence in the full text. Try to "getByRole("heading", { name: "Package installation failed" }) to see what I mean. Or simply add a the prop title and test how the dialog looks.

Reply:
>  having a title make sense in many cases (all?)

Yes, all dialoga should have good titles

> Because extending the Questions API is not trivial (do not forget that questions can be handled using command line interface) I'll create a separate card for that.

So, let's do the temporary solution more robust by adding a specialized question component. Or goes por the implementation proposed by @imobachgs and add missing data as part of the mentioned hash data instead of gluing all in the text field.